src/modules/plus/filter_gradientmap.yml

```diff
@@ -0,0 +1,32 @@
+schema_version: 0.1
+type: filter
+identifier: gradientmap
+title: Gradient Map
+version: 1
+copyright: Martin Rodriguez Reboredo
+creator: Martin Rodriguez Reboredo
+license: LGPLv2.1
+language: en
+tags:
+  - Video
+description:
+  Maps the colors of an image to a gradient according to their intensity.
+
+parameters:
+  - identifier: color.*
+    title: Color Stop
+    type: string
+    description: |
+      The gradient color stops.
+
+      A set of pairs that each describe a point in the gradient. A pair consists
+      on a color and a position with a separator in between them.
+
+      By default, the filter has a gradient that goes from black to white.
+
+          color.1='0xff000000 0.0' color.2='0xffffffff 1.0'
```

Comment:
CSS style color specifications are now preferred in examples, but that is not a required change. I leave the choice to you.